src/app/dashboard-widgets/pipelines-widget/pipelines-widget.component.html

```diff
@@ -6,7 +6,7 @@ <h2 class="card-pf-title">
       </h2>
     </div>
     <div class="card-pf-body f8-card-body">
-      <div class="f8-blank-slate-card" *ngIf="!(buildConfigsCount | async)">
+      <div class="f8-blank-slate-card" *ngIf="1 > buildConfigsCount; else appsAvailable">
```

Comment:
`1 > buildConfigsCount`? This is a bit awkward to read... why not `buildConfigsCount < 1` or `buildConfigsCount === 0` even?